src/test/java/com/ibm/watson/developer_cloud/service/GenericServiceTest.java

```diff
@@ -238,5 +301,51 @@ public void testCustomUserAgent() throws InterruptedException {
     assertTrue(request.getHeader(HttpHeaders.USER_AGENT).endsWith("foo-bar"));
     service.setDefaultHeaders(null);
   }
-  
+
+  /**
+   * Tests request without authentication.
+   */
+  @Test(expected = IllegalArgumentException.class)
+  public void testNoAuthentication() {
+    new SpeechToText().getModels();
+  }
+
+  /**
+   * Tests request with skipped authentication
+   * @throws InterruptedException the interrupted exception
+   */
+  @Test
+  public void testSkipAuthentication() throws InterruptedException {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://www.codacy.com/app/watson-developer-cloud/java-sdk/file/2722166938/issues/source?bid=3303699&fileBranchId=3347904#l318)